src/mod.ts

```diff
@@ -14,7 +14,7 @@ export const run = async function (functionDir: string) {
   // For the hosted runtime, we only support js/ts files named w/ the callback_id
   // They should already be bundled into single files as part of the package uploaded
   const resp = await DispatchPayload(payload, (functionCallbackId) => {
-    const supportedExts = ["js", "ts"];
+    const supportedExts = ["js"];
```

Comment:
If we don't expect to support multiple different filetypes anymore, then let's remove most of the code here: no point having an array of extensions, or looping over them, to return an array of files.